CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## Release 0.7.0
+
+New features:
+- Track short-lived connections
+  [#356](https://github.com/weaveworks/scope/issues/356)
+- Details pane: add a lot more information and present it in a more intuitive way.
```

Comment:
Should add more info here - what information has been added?  How has the presentation changed?